src/platform/forms-system/src/js/utilities/data/reduceErrors.js

```diff
@@ -137,30 +240,37 @@ export const getPropertyInfo = (pageList = [], name, instance = '') => {
  * @param {Form~pageList} pageList - list of all form pages from `route.pageList`
  * @return {Form~errors} - Finely curated list of form errors
  */
-export const reduceErrors = (errors, pageList) =>
+export const reduceErrors = (errors, pageList, reviewErrors = {}) =>
   errors.reduce((processedErrors, error) => {
+    let errorIndex = null; // save key (index) of array items with __error
     const findErrors = (name, err) => {
       if (err && typeof err === 'object') {
         // process the last type of error message which provides an `__errors`
         // message array. If there are multiple errors, we'll join them into
         // one message.
         if (
-          err?.__errors?.length &&
-          !errorExists(processedErrors, name, null)
+          err.__errors?.length &&
+          !errorExists(processedErrors, name, errorIndex)
         ) {
           const { chapterKey = '', pageKey = '' } = getPropertyInfo(
             pageList,
             name,
           );
-          processedErrors.push({
-            name,
-            index: null,
-            message: err.__errors.join('. '),
-            chapterKey,
-            pageKey,
-          });
-        }
-        if (err.property) {
+          // message returns null if we don't want a link to show up
+          // happens for 526's when a new disability is missing (has error), and
+          // the nested required condition (also has an error) is also missing
```

Comment:
```suggestion
          // `message` is null if we don't want a link to show up.
          // For example, this happens for the 526 when a new disability is
          // missing (has error), and the nested required condition (also has
          // an error) is also missing.
```
```suggestion
          // message returns null if we don't want a link to show up
          // happens for 526's when a new disability is missing (has error), and
          // the nested required condition (also has an error) is also missing
```

Reply:
I suspect you wanted the first suggested change to be applied? Because this is what I changed it to:

```
// `message` is null if we don't want a link to show up.
// For example, this happens for the 526 when a new disability is
// missing (has error), and the nested required condition (also has
// an error) is also missing.
```